src/media/js/routes.js

```diff
@@ -34,6 +34,11 @@ define('routes',
         {'pattern': '^/usage$', 'view_name': 'usage'},
     ]);
 
+    var search = '/api/v2/fireplace/search/?cache=1&vary=0';
+    if (settings.meowEnabled) {
+        search = '/api/v2/multi-search/?cache=1&vary=0';
```

Comment:
The way I envisioned it was just to have different endpoints. I can certainly make the API handle a content type param, but AFAIK there is no bug for it as that wasn't the original plan.